src/components/binding/binding-loader.js

```diff
@@ -98,7 +98,7 @@ genotet.BindingLoader.prototype.loadFullTrack = function(trackIndex, gene,
     this.signal('loadComplete');
     if (addTrack) {
       // Add one more track.
-      this.signal('track');
+      this.loadBindingList();
```

Comment:
Do we really need to reload the binding list every time a track is added? The listed data is unchanged (unless the user uploads a new data). I think only re-drawing the panel is necessary without involving server for the listing.

Maybe you can load the binding data list once and for all, on system start (e.g. genotet.data.init).